rasa/core/constants.py

```diff
@@ -26,3 +26,17 @@
 ACTION_NAME_SENDER_ID_CONNECTOR_STR = "__sender_id:"
 
 BEARER_TOKEN_PREFIX = "Bearer "
+
+# the lowest priority intended to be used by machine learning policies
+DEFAULT_POLICY_PRIORITY = 1
+# the priority intended to be used by mapping policy
```

Comment:
```suggestion
# the priority intended to be used by `MappingPolicy`
```

same for the others

Reply:
I wanted it to be general, e.g. fallback is used by two policies

Reply:
memo as well, etc

Reply:
Ok, but how about the others?

Reply:
what if people want to use these priorities? Also, I think it's better to keep one style for all

Reply:
memo for two as well, etc. This priorities are not class specific